elements/glossy-portfolio-theme/lib/glossy-portfolio-grid.js

```diff
@@ -25,8 +25,50 @@ export class GlossyPortfolioGrid extends DDDSuper(I18NMixin(LitElement)) {
 
     this.filtersList = [],
     this.filteredData = [];
-    this.data = [];
     this.activeFilter = '';
+    this.__disposer = this.__disposer || [];
+
+    
+    // get children, related content, or prev and next pages
+    autorun((reaction) => {
+      const activeItem = toJS(store.activeItem);
+      
+      if (activeItem) {
+        this.activeItem = activeItem;
+        // find parent of activeItem
+        this.activeParent = store.manifest.items.find((d) => activeItem.parent === d.id)||"";
```

Comment:
I think there's an easier way of doing this like `store.loadItemById` or something and you can just pass the `item.parent` to it